pylint/checkers/spelling.py

```diff
@@ -21,7 +21,6 @@
 
```

Comment:
The stylistic changes to `pylint/checkers/spelling.py` are unrelated to supporting isort 5. Wouldn't it be better to move these into a separate PR?

The same goes for

- `pylint/checkers/base.py`
- `pylint/checkers/spelling.py`
- `pylint/checkers/utils.py`
- `pylint/epylint.py`
- `pylint/extensions/docparams.py`
- `pylint/path.py`
- `setup.py`
- `tests/checkers/unittest_typecheck.py`
- `tests/extensions/test_bad_builtin.py`
- `tests/extensions/test_broad_try_clause.py`
- `tests/extensions/test_check_docs_utils.py`
- `tests/extensions/test_check_mccabe.py`
- `tests/extensions/test_elseif_used.py`
- `tests/extensions/test_emptystring.py`
- `tests/extensions/test_redefined.py`
- `tests/lint/unittest_lint.py`
- `tests/test_import_graph.py`
- `tests/test_regr.py`
- `tests/unittest_pyreverse_diadefs.py`
- `tests/unittest_pyreverse_inspector.py`

Reply:
These changes are indeed not related to supporting both versions 4 and 5 of isort.

However, pylint's own code should follow what isort 5 mandates, and that's what these changes are. The corresponding commit says so: ` Switch to isort 5 for pylint's own code`. (This pull request has 3 separate commits, which should not be squashed together.)

Reply:
Ok fair enough. I guess it's an ok trade-off against the fact that the PR becomes a bit heavy to review. :+1:

Reply:
> Ok fair enough. I guess it's an ok trade-off against the fact that the PR becomes a bit heavy to review. 👍
When MRs include these types of changes, reviewing commit by commit is usually easier.  Cheers!

Reply:
Yeah we're kinda forced to do that because of the formatting tests that won't pass otherwise. We're also forced to break atomicity of commits (this one commit probably break tests, or if it does not the isort 5 commit would break tests).  Still better than mixing up the changes.